glide.yaml

```diff
@@ -19,14 +19,25 @@ import:
   - pkg/api
   - pkg/api/install
   - pkg/api/v1
+  - pkg/apis/batch/v1
   - pkg/apis/extensions/install
   - pkg/apis/extensions/v1beta1
-  - kubernetes/typed/admissionregistration/v1alpha1
 - package: k8s.io/apimachinery
+  version: 1fd2e63a9a370677308a42f24fd40c86438afddf
   subpackages:
   - pkg/api/resource
   - pkg/apis/meta/v1
   - pkg/runtime
   - pkg/util/intstr
 - package: github.com/sirupsen/logrus
   version: v1.0.3
+- package: github.com/fsouza/go-dockerclient
+  version: d6c9513c3abe61f13e41c22bc013346eb96e1fe5
```

Comment:
can we use this in `3f9370a4738ba8d0ed6eea63582ca6db5eb48032` version?

The same version is used by OpenShift -  this will make vendoring easier

Reply:
done!